applications/altran-v2x/readme.md

```diff
@@ -0,0 +1,74 @@
+```
+SPDX-License-Identifier: Apache-2.0
+Copyright (c) 2021 Altran Technology
+```
+
+Overview of Altran V2X Apps
+############################
```

Comment:
File does not seem to be rendered properly.
https://github.com/open-ness/edgeapps/blob/d8ee8dae6d03a4c3355815b3e6141b27680b6e92/applications/altran-v2x/readme.md

Correct syntax:
```
# Overview of Altran V2X Apps
```